config/datatypes_conf.xml.sample

```diff
@@ -736,6 +736,7 @@
     <datatype extension="dada2_mergepairs" type="galaxy.datatypes.binary:RData" subclass="true" display_in_upload="true" />
     <datatype extension="dada2_sequencetable" type="galaxy.datatypes.tabular:Tabular" mimetype="application/text" subclass="true" display_in_upload="true" />
     <datatype extension="dada2_uniques" type="galaxy.datatypes.tabular:Tabular" mimetype="application/text" subclass="true" display_in_upload="true" />
+    <datatype extension="ckpt" type="galaxy.datatypes.binary:Tensorflow" subclass="true" display_in_upload="true" />
```

Comment:
```suggestion
    <datatype extension="ckpt" type="galaxy.datatypes.binary:Binary" subclass="true" display_in_upload="true" />
```

and remove the changes to lib/galaxy/datatypes/binary.py

Sorry my first comment wasn't very clear.